vent/controller/control_module.py

```diff
@@ -240,12 +233,16 @@ def update(self, pressure):
 
 class ControlModuleSimulator(ControlModuleBase):
     # Implement ControlModuleBase functions
-    def __init__(self):
-        super().__init__()  # get all from parent
+    def __init__(self, threadID, name):
+
+        ControlModuleBase.__init__(self)
+        threading.Thread.__init__(self)
```

Comment:
Maybe you also need to clean up these threading base class related code?

Reply:
done